plugins/woocommerce/client/blocks/assets/js/blocks/checkout/inner-blocks/checkout-terms-block/edit.tsx

```diff
@@ -145,10 +145,9 @@ export const Edit = ( {
 			>
 				{ checkbox ? (
 					<>
-						<CheckboxControl
-							id="terms-condition"
```

Comment:
I think we might need to keep this id for people using it in their customizations.